[docs] Show how to customize drawing area background (#19682)
mui/mui-x: docs/data/charts/styling/styling.md

{{"demo": "SxStyling.js"}}

### Drawing area background

To set a background color in the drawing area, you should create a dedicated `<rect />`.

> This is only doable with [composition](/x/react-charts/composition/) because you have to place this new component before all plot component.

Reviewer: ```suggestion
This is only doable with [composition](/x/react-charts/composition/) because you have to place this new component before all plot components.
```

> The following demo defines a basic `<Background />` component to add light gray background.

Reviewer: ```suggestion
The following demo defines a basic `<Background />` component that adds a light gray background.
```
